packages/runtime/test-runtime-utils/src/insecureUrlResolver.ts

```diff
@@ -88,11 +88,20 @@ export class InsecureUrlResolver implements IUrlResolver {
         }
     }
 
-    private resolveHelper(documentId: string) {
+    private resolveHelper(documentId: string, uri: string) {
         const encodedTenantId = encodeURIComponent(this.tenantId);
         const encodedDocId = encodeURIComponent(documentId);
+        let documentUrl: string;
+        if (uri !== "") {
+            const pathArray = uri.split("/");
+            const encodedPathArray = pathArray.map((pathPart) => encodeURIComponent(pathPart));
```

Comment:
>const encodedPathArray = pathArray.map((pathPart) => encodeURIComponent(pathPart)); [](start = 12, length = 83)

Do we need to encode the parts of the URL other than the documentId?
When requesting a component from the Container, we do not encode the component Ids. So I would think this is not required.